nixos/modules/services/x11/neolight.nix

```
@@ -0,0 +1,25 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.xserver.xkb.neolight;
+in
+{
+  # interface
+  options.services.xserver.xkb.neolight = {
+    package = lib.mkPackageOption pkgs "neolight" { };
+    enable = lib.mkEnableOption "Enable neolight xkb layout, extra keyboard layers for programming based on Neo";
```

```suggestion
    enable = lib.mkEnableOption "neolight xkb layout, extra keyboard layers for programming based on Neo";
```
`mkEnableOption` already prepends `Whether to enable `, so your description would be generated as "Whether to enable Enable ..."